templates/stats/components/user-stats.html

```diff
@@ -1 +1,96 @@
-<h1>user</h1>
\ No newline at end of file
+{% load static i18n %}
+<link rel="stylesheet" href="{% static 'css/sections/stats.css' %}">
+
+<div class="container-fluid">
+	<!-- Empty Space Row -->
+	<div class="row mt-2"></div>
+  
+	<!-- Profile Header Section -->
+	<div class="row profile-header">
+	  <!-- Empty Space Column -->
+	  <div class="col-0 col-md-1"></div>
+	  
+	  <!-- First Column -->
+	  <div class="col-6 col-md-4">
+		<!-- Distance Row -->
+		<div id="rectangle" class="row">
+			<div class="row mt-3"></div>
+			<!-- Image + Words Row -->
+			<div class="row d-flex align-items-justified">
+				<div class="col-0 col-md-3"></div>
+				<div class="col-0 col-md-1">
+					<img style="max-height: 80px; width: auto;" src="{% static 'achievements/icon-distance.png' %}" alt="{% translate 'Distance icon' %}" title="{% translate 'Distance walked together' %}">
+				</div>
+				<div class="col-0 col-md-6">
+					<p id="userStatsMax">{{ distance }}</p>
+				</div>
+			</div>
+			<p id="userStatsText">{% translate "DISTANCE WALKED TOGETHER" %}</p>
+		</div>
+		<!-- Empty Row -->
+		<div class="row mt-5"></div>
+		<!-- High Five Row -->
+		<div id="rectangle" class="row">
+			<div class="row mt-3"></div>
+			<!-- Image + Words Row -->
+			<div class="row d-flex align-items-justified">
+				<div class="col-0 col-md-3"></div>
+				<div class="col-0 col-md-1">
+					<img style="max-height: 80px; width: auto;" src="{% static 'achievements/icon-friends.png' %}" alt="{% translate 'Distance icon' %}" title="{% translate 'Distance walked together' %}">
+				</div>
+				<div class="col-0 col-md-6">
+					<p id="userStatsMax">{{ high_five }}</p>
+				</div>
+			</div>
+			<p id="userStatsText">{% translate "HIGH FIVE COUNT" %}</p>
+		</div>
+	  </div>
+	  <!-- Bff Column -->
+	  <div class="col-2 col-md-2 d-flex flex-column align-items-center justify-content-center">
+		<div class="bffText">B F F</div>
+		<div class="bffImage">
+			<img class="roundedImage" src="{% static 'images/roaraujo.jpeg' %}" alt="BFF">
+		</div>
+		<p id="userStatsBFF">{% translate "you played " %} {{ bff_matches }} {% translate "matches with " %} {{ bff_login }}!</p>
```

Comment:
When translating a mix of text with variables, it's generally good practice to use one single `blocktrans` tag instead of separate `translate` tags, like so:

```
      <p id="userStatsBFF">
        {% blocktrans %}You played {{ bff_matches }} matches with {{ bff_login }}!{% endblocktrans %}
      </p>
```

That is because it allows for accurate translations for all types of languages, especially the ones that do not follow the "subject - verb - object" sentence order, like Japanese or Arabic. Our application will be translated only into Portuguese and French, so the way you did it will most likely be OK! So it's up to you to decide whether to make this change or not - either way will work alright.

Reply:
I made the change, didn't knew this existed! Thanks!